### PR TITLE
Fixed #109 - sys_write()

### LINKED ISSUE #109: Syscall write() broken on qemu
`sys_write()` returns the bytesize of the bytecounter of the written data, instead of the bytes written.
So it always returns 8, because length is a `ssize_t`.

https://github.com/hermitcore/libhermit/blob/423cb0e80e4b81122fb088fdbc5bb385d9232933/kernel/syscall.c#L285

### PR DESCRIPTION
**Pull request by pdressen**

### CODE REVIEW

File: kernel/syscall.c [hunk 1]
```diff
@@ -282,7 +282,13 @@ ssize_t sys_write(int fd, const char* buf, size_t len)
 	}
 
 	if (fd > 2)
-		i = socket_recv(s, &i, sizeof(i));
+	{
```
Comment (jounathaen) [inline kernel/syscall.c]: Your if-braces do not quite match the rest of the file

Comment (pdressen) [reply kernel/syscall.c]: In this function alone, there are two different styles.
Which is the preferred one?

Comment (jounathaen) [reply kernel/syscall.c]: Oh true. Nevermind. Sorry for bothering

**Comment by stlankes** [comment]
Thanks! This was a stupid failure...

**Pull request merged**